src/esssans/beam_center_finder.py

```diff
@@ -68,10 +86,16 @@ def beam_center_from_center_of_mass(
         The beam center position as a vector.
     """
 
-    summed = data.sum(list(set(data.dims) - set(data.coords['position'].dims)))
+    # TODO: the flattening is required for the selection with a boolean mask to work.
+    # Until slicing with a multi-dimensional mask is supported in Scipp, we could avoid
+    # the copy made here by working with the underlying numpy arrays.
```

Comment:
It is not going to be supported in Scipp, since it is conceptually impossible (without automatic flattening, which would defeat the purpose of Scipp).

Reply:
Ok, removed comment.